src/sentry/api/serializers/models/organization.py

```diff
@@ -509,6 +512,19 @@ def _team_list(self, organization: Organization, access: Access) -> list[Team]:
 
         return team_list
 
+    def _team_memberships(self, access: Access) -> List[Team]:
+        if access.api_member is None:
+            return []
+
+        return [
+            omt.team
+            for omt in OrganizationMemberTeam.objects.filter(
+                organizationmember_id=access.api_member.id,
+                is_active=True,
+                team__status=TeamStatus.VISIBLE,
+            ).select_related("team")
```

Comment:
Are you not able to use `access.api_team_members` for this?

Reply:
Yup, I can.

Reply:
Fixed